test/support/utils.go

```diff
@@ -39,3 +39,7 @@ func WriteToOutputDir(t Test, fileName string, fileType OutputType, data []byte)
 	t.Expect(os.WriteFile(path.Join(t.OutputDir(), fileName+"."+string(fileType)), data, fs.ModePerm)).
 		To(gomega.Succeed())
 }
+
+func Int32(v int32) *int32 {
```

Comment:
https://github.com/project-codeflare/codeflare-operator/blob/2ddc14f16c9ab7a5553b5fb8af922430e94f234c/test/support/utils.go#L27 can be used instead.

Reply:
Thank you!